pyrevitlib/pyrevit/__init__.py

```diff
@@ -56,7 +56,7 @@
 
 
 # try get net folder
-net_folder = "netcore" if compat.is_netcore() else "netfx"
+net_folder = "netcore" if compat.NETCORE else "netfx"
```

Comment:
Check the merge conflict, rename `net_folder` to `DOTNET_RUNTIME_ID` and remove the lines above.
Not sure if the more verbose check we do is needed and should be ported to the `compat.NETCORE` (I believe it doesn't)

Reply:
@sanzoghenzo `compat.NETCORE` point out the major version greater or equal 8 is enough since Revit 2025 start with NET 8

Reply:
Please do the opposite, use `DOTNET_RUNTIME_ID` instead of `net_folder`.

It's ok to check for the version 8; ehat I meant is that the current version also checks for `".NET Core" in SRI.RuntimeInformation.FrameworkDescription`, but I believe it is ok to remove it

Reply:
@sanzoghenzo I've got it, thank you